src/aerie_cli/aerie_client.py

```diff
@@ -1611,3 +1611,58 @@ def get_sequence_json(self, command_dictionary_id: int, edsl_body: str) -> dict:
             body=edsl_body
         )
         return resp["seqJson"]
+
+    def get_metadata(self) -> list:
+        """Get metatdata
+
+        Returns:
+            list: a list of the metadata keys and schemas
+        """
+        get_metadata_query = """
+        query GetMetadata {
+            activity_directive_metadata_schema {
+                key
+                schema
+            }
+        }
+        """
+
+        resp = self.host_session.post_to_graphql(get_metadata_query)
+        return resp
+
+    def add_metadata_schemas(self, schemas) -> list:
+        """Add metadata schemas
+        """
```

Comment:
Would be helpful to document the fields necessary in the `schemas` argument (ideally add a type hint, too).